pymeasure/instruments/agilent/agilent33500.py

```diff
@@ -48,6 +46,317 @@
+        """ A boolean property that controls whether the burst mode is on
+        (True) or off (False). Can be set. """,
+        validator=strict_discrete_set,
+        map_values=True,
+        values={True: 1, False: 0},
+    )
+
+    burst_mode = Instrument.control(
+        "SOUR{ch}:BURS:MODE?",
+        "SOUR{ch}:BURS:MODE %s",
+        """ A string property that controls the burst mode. Valid values
+        are: TRIG<GERED>, GAT<ED>. This setting can be set. """,
+        validator=strict_discrete_set,
+        values=["TRIG", "TRIGGERED", "GAT", "GATED"],
+    )
+
+    burst_period = Instrument.control(
+        "SOUR{ch}:BURS:INT:PER?",
+        "SOUR{ch}:BURS:INT:PER %e",
+        """ A floating point property that controls the period of subsequent bursts.
+        Has to follow the equation burst_period > (burst_ncycles / frequency) + 1 µs.
+        Valid values are 1 µs to 8000 s. Can be set. """,
+        validator=strict_range,
+        values=[1e-6, 8000],
+    )
+
+    burst_ncycles = Instrument.control(
+        "SOUR{ch}:BURS:NCYC?",
+        "SOUR{ch}:BURS:NCYC %d",
+        """ An integer property that sets the number of cycles to be output
+        when a burst is triggered. Valid values are 1 to 100000. This can be
+        set. """,
+        validator=strict_range,
+        values=range(1, 100000),
+    )
+
+    arb_file = Instrument.control(
+        "SOUR{ch}:FUNC:ARB?",
+        "SOUR{ch}:FUNC:ARB %s",
+        """ A string property that selects the arbitrary signal from the volatile
+        memory of the device. String has to match an existing arb signal in volatile
+        memore (set by data_arb()). Can be set. """,
+    )
+
+    arb_advance = Instrument.control(
+        "SOUR{ch}:FUNC:ARB:ADV?",
+        "SOUR{ch}:FUNC:ARB:ADV %s",
+        """ A string property that selects how the device advances from data point
+        to data point. Can be set to 'TRIG<GER>' or 'SRAT<E>' (default). """,
+        validator=strict_discrete_set,
+        values=["TRIG", "TRIGGER", "SRAT", "SRATE"],
+    )
+
+    arb_filter = Instrument.control(
+        "SOUR{ch}:FUNC:ARB:FILT?",
+        "SOUR{ch}:FUNC:ARB:FILT %s",
+        """ A string property that selects the filter setting for arbitrary signals.
+        Can be set to 'NORM<AL>', 'STEP' and 'OFF'. """,
+        validator=strict_discrete_set,
+        values=["NORM", "NORMAL", "STEP", "OFF"],
+    )
+
+    arb_srate = Instrument.control(
+        "SOUR{ch}:FUNC:ARB:SRAT?",
+        "SOUR{ch}:FUNC:ARB:SRAT %f",
+        """ An floating point property that sets the sample rate of the currently selected
+        arbitrary signal. Valid values are 1 µSa/s to 250 MSa/s (maximum range, can be lower
+        depending on your device). This can be set. """,
+        validator=strict_range,
+        values=[1e-6, 250e6],
+    )
+
+    def data_volatile_clear(self):
+        """
+        Clear all arbitrary signals from the volatile memory. This should be done if the same name
```

Comment:
I think docstring should have a single line description and
 a more detailed description after a blank line.
Same comment for other docstring.

Reply:
Re-formatted all dosctrings with that issue (I hope) in https://github.com/pymeasure/pymeasure/pull/763/commits/3e71c1eee00e7ff48596b6418855cf6a67fe831e